index.html

```diff
@@ -32,7 +32,139 @@ <h2 class="first-name software-dev">I'm a software developer</h2>
+  <div class="multi-post">
+    <h3 class="multi-text-color">Multi-Post Stories Gain+Glory</h3>
+  <ul class="language display-flex">
+    <li class="language-background">Ruby on rails</li>
+    <li class="language-background">css</li>
+    <li class="language-background" >JavaScript</li>
+    <li class="language-background">html</li>    
+  </ul>
+  <button class="see-project">See Project</button>
+</div>
+</div>
+
+<div class="project-1 grid-project display-flex">
+  <div class="gray-container"></div>  
+  <div class="multi-post">
+    <h3 class="multi-text-color">Multi-Post Stories Gain+Glory</h3>
+  <ul class="language display-flex">
+    <li class="language-background">Ruby on rails</li>
+    <li class="language-background">css</li>
+    <li class="language-background" >JavaScript</li>
+    <li class="language-background">html</li>    
+  </ul>
+  <button class="see-project">See Project</button>
+</div>
+</div>
+</section>
+
+<!-- About me Session -->
+<section class="about-container">
+  <h2 class="about-me">About Me</h2>
+     <p class="about-message"> I can help you build a product, feature or website 
+         Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don't hestiate to contact me.</p>
+  <button class="see-project add-bottom-margin">Get My Resume</button>  
+</section>
+
+<!-- Language Session -->
+<section class="grid-frame-skill">
+<div class="grid-child-frame-skill">
+  <h2 class="top-padding">Languages</h2>
+  <ul class="language buttom-pad display-flex">
+    <li class="frame-skill-color frame-skill-background">JavaScript</li>
+    <li class="frame-skill-color frame-skill-background">Ruby</li>
+    <li class="frame-skill-color frame-skill-background">CSS</li>
+    <li class="frame-skill-color frame-skill-background">Html</li>    
+  </ul>
+</div>
+<!-- Frameworks Session -->
+<div class="grid-child-frame-skill">
+  <h2 class="top-padding">Frameworks</h2>
+  <ul class="language buttom-pad display-flex">
+    <li class="frame-skill-color frame-skill-background">Bootstrap</li>
+    <li class="column-gap-frame frame-skill-color frame-skill-background">Ruby on rails</li>
+    <li class="frame-skill-color frame-skill-background">Rspec</li>
+    <li class="frame-skill-color frame-skill-background">Capybara</li>
+    <li class="frame-skill-color frame-skill-background">Selenium</li>     
+  </ul>
+</div>
+<!-- Skill Session -->
+<div class="grid-child-frame-skill">
+  <h2 class="top-padding">Skills</h2>
+  <ul class="language buttom-pad display-flex">
+    <li class="frame-skill-color frame-skill-background">Codekit</li>
+    <li class="frame-skill-color frame-skill-background">GitHub</li>
+    <li class="frame-skill-color frame-skill-background">Codepen</li>
+    <li class="frame-skill-color frame-skill-background">Gitlab</li>
+    <li class="frame-skill-color frame-skill-background">Terminal</li>     
+  </ul>
+</div>
+</section>
```

Comment:
- [ ] Kindly ensure you follow the design as we have on the template. Each of your categories is missing the accompanying image before the texts, Kindly add the images.

- [ ] Your tech stacks are somewhat aligned to the right, whereas on the template they are centered. Kindly fix that.

- [ ] Kindly ensure you use the appropriate background color as we have on the template.

- [ ] Kindly ensure you implement the appropriate spaces around the edges and the bottom of the containers.

  Your Implementation | Figma Design
  :---:|:---:
  ![ezea-lang](https://user-images.githubusercontent.com/36542986/164949410-a82356a1-440d-4dc5-8841-f0d7650f942f.jpg) | ![ezea-lang-figma](https://user-images.githubusercontent.com/36542986/164949407-f04cdc8d-0159-40cd-ac7a-54492f889d77.jpg)

Reply:
good observation.

Reply:
good

Reply:
good, thanks